_posts/2015-10-21-Onefootbot.md

```diff
@@ -0,0 +1,123 @@
+[Onefootball](https://www.onefootball.com/) everything is around football we wrote a small hubot script that would listen
+to the word football in all of our open chat rooms and respond with a random football quote.
+
+The main functionality of the script looks like this:
+
+```coffee
+  robot.hear /football/i, (res) ->
+    res.send getRandomFootballQuote()
+
+  getRandomFootballQuote = () ->
+    quotes = [
+      'Football is the ballet of the masses. - Dmitri Dmitriyevich Shostakovich',
+      'Some people tell me that we professional players are soccer slaves. Well, if this is slavery, give me a life sentence. - Sir “Bobby” Charlton',
+      'In his life, a man can change wives, political parties or religions but he cannot change his favourite soccer team. - Eduardo Hughes Galeano',
+      'You don’t have to have been a horse to be a jockey. - Arrigo Sacchi',
+      'If you’re attacking, you don’t get as tired as when you’re chasing. - Kyle Rote, Jr.',
+      'I don’t believe skill was, or ever will be, the result of coaches. It is a result of a love affair between the child and the ball. - Roy Keane',
+      'Soccer riots kill, at most, tens. Intellectuals’ ideological riots sometimes kill millions. - John Jay McCarthy',
+      'To say that these men paid their shillings to watch twenty-two hirelings kick a ball is merely to say that a violin is wood and catgut, and that Hamlet is so much paper and ink. - John Boynton Priestley',
+      'The ball is round, the game lasts ninety minutes, and everything else is just theory. - Josef “Sepp” Herberger',
+      'International football is the continuation of war by other means. - George Orwell',
+      'In football everything is complicated by the presence of the opposite team. - Jean-Paul Sartre',
+      'In football, the worst blindness is only seeing the ball. - Nelson Falcão Rodrigues',
+      'The first 90 minutes are the most important. - Sir “Bobby” Robson',
+      'A football team is like a beautiful woman. When you do not tell her, she forgets she is beautiful. - Arsène Wenger',
+      'The trouble with referees is that they know the rules, but they do not know the game. - William “Bill” Shankly',
+      'It is better to win ten times 1-0 than to win once 10-0. - Vahid “Vaha” Halilhodžic',
+      'Football is a simple game; 22 men chase a ball for 90 minutes and at the end, the Germans win. - Gary Lineker',
+      'In Europe, it’s different – you eat soccer, you breathe soccer, you drink soccer. Everything is about soccer. - Thomas Dooley',
+      'We must have had 99 percent of the game. It was the other three percent that cost us the match. - Ruud Gullit',
+      'Sometimes in soccer you have to score goals.- Thierry Henry',
+      'At the World Cup I watched the game Austria against Cameroon. On the one side exotics, alien culture , wild rites - and on the other side : Cameroon ! - Dieter Nuhr',
+      'The subjunctive is the enemy of the losers. - Jens Lehmann'
+    ]
```

Comment:
Do we need that whole big array for demo purposes? We could alternatively be fine with 2, 3 elements inside.